scripts/nginx/requestrr.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Nginx Configuration for requestrr
+#shellcheck source=sources/functions/users
+. /etc/swizzin/sources/functions/users
+master=$(_get_master_username)
+
+if [[ ! -f /etc/nginx/apps/requestrr.conf ]]; then
+    cat > /etc/nginx/apps/requestrr.conf << SRC
+location ^~ /requestrr {
+  proxy_pass        http://127.0.0.1:4545/requestrr;
+  proxy_set_header Host \$proxy_host;
+  proxy_set_header X-Forwarded-For \$proxy_add_x_forwarded_for;
+  proxy_set_header X-Forwarded-Proto \$scheme;
+  proxy_redirect off;
+  # Basic Auth if Wanted
+  ### This shouldn't be needed
+  ### auth_basic "What's the password?";
+  ### auth_basic_user_file /etc/htpasswd.d/htpasswd.${master};
```

Comment:
I would actually say that if there's only one account that can use this, might as well not set up app-side auth for the nginx-served app. Just like sonarr, just do basic auth and unprotect the app.

Reply:
The app enforces it's own protection.